etc/qbittorrent.profile

```diff
@@ -34,11 +34,12 @@ nogroups
 nonewprivs
 noroot
 nosound
-protocol unix,inet,inet6
+protocol unix,inet,inet6,netlink
```

Comment:
I'm not sure. I'm not using much networking features under calibre but it has options like running it's own server which can be affected by his.

I think we shouldn't block by default anything what applications try access by default and throwing errors as it's very possible that it break some usage which can be hard to debug.